contessa/rules.py

```diff
@@ -89,16 +90,17 @@ def apply(
                         raise ValueError(
                             f"Your query for rule `{self.name}` of type `{self.type}` does not return list of booleans in column `valid`."
                         )
-                    if row[0]:
+                    total += 1
+                    if row[0] is True:
```

Comment:
I would go for : if row[0], if not row[0]

Reply:
`if not row[0]` would match all falsey values (e.g. None) which I do not want.